source/Grid/Grid.js

```diff
@@ -551,7 +558,8 @@ export default class Grid extends Component {
               height: totalRowsHeight,
               maxWidth: totalColumnsWidth,
               maxHeight: totalRowsHeight,
-              pointerEvents: isScrolling ? 'none' : 'auto'
+              pointerEvents: isScrolling ? 'none' : 'auto',
+              position: 'relative'
```

Comment:
Static styles (like `position`) should go in `styles.css` to be more consistent with other styling stuff.